packages/cli-dxns/src/dxos.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package dxos;
+
+message DXNS_cid {
+  bytes data = 1;
+}
+
+message IPFS_cid {
+  bytes data = 1;
+}
+
```

Comment:
`IPFS_CID`

Reply:
It would be nice to have TS like types, but I don't think wrapping these in messages is right. So unfortunately, I would just have naked bytes as the type -- unless there's a reason to have another field here?

Reply:
The idea is that tools will be able to introspect the schema and deduce that the field is not just a sequence of bytes but a link to another record: for example blockchain explorer inserting hyperlinks for those ipfs fields